src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">
+        {comment.name}
+      </strong>
+
+      {' by '}
+
+      <a
+        className="CommentInfo__email"
+        href={`mailto:${comment.email}`}
+      >
+        {comment.email}
+      </a>
+    </div>
+
+    <div className="CommentInfo__body">
+      {comment.body}
+    </div>
```

Comment:
Why you don't use a text tag for the body? Like `<p>`? Think about what tag will be better for the `comment` text body